docusaurus/sidebars-ios.json

```diff
@@ -1,49 +1,453 @@
 {
-    "docs": [
-      "home",
-      "introduction",
-      {
-        "Guides": [
-          "guides/integration",
-          "guides/getting-started",
-          "guides/working-with-channel-list",
-          "guides/working-with-a-channel",
-          "guides/working-with-messages",
-          "guides/working-with-user",
-          "guides/pinned-messages",
-          "guides/working-with-attachments",
-          "guides/working-with-reactions",
-          "guides/working-with-members",
-          "guides/working-with-watchers",
-          "guides/events",
-          "guides/push-notifications",
-          "guides/connection-status",
-          "guides/ui-customization",
-          "guides/moderation",
-          "guides/filter-query-guide",
-          "guides/multi-tenancy",
-          "guides/debugging"
-        ]
-      },
-      { "StreamChat Controllers":
-        [
-          {
-           "type": "autogenerated",
-           "dirName": "controllers"
-          }
-         ] 
-       },
-      
-      { "UI Components": 
-        [
-         {
-          "type": "autogenerated",
-          "dirName": "ui-components"
-         }
-        ] 
-      },
-      "migrating-from-1.x-and-2.x",
-      "faq",
-      "glossary"
-    ]
-  }
+  "docs": [
+    "home",
+    "introduction",
+    {
+      "Guides": [
+        {
+          "Getting Started": [
+            "guides/getting-started",
+            "guides/integration"
+          ]
+        },
+        {
+          "General": [
```

Comment:
Better to not have a group for these, "general" doesn't cut it

Reply:
Done.

Reply:
I agree. It would be better to treat a guide being General as implicit if it isn't grouped with other guides.

Would it make sense to arrange these in a way similar to what [React Native](https://staging.getstream.io/chat/docs/sdk/reactnative/?language=javascript) is currently doing with their TOC?